src/xcode/ENA/ENA/Source/Scenes/Events/Cells/EventCellModel.swift

```diff
@@ -18,7 +18,8 @@ protocol EventCellModel {
 
 	var title: String { get }
 	var address: String { get }
-
+	var titlePrefix: String { get }
```

Comment:
```suggestion
	var titleAccessiblityLabel: String { get }
```

I think this is not really a prefix as it also contains the title itself. Just a suggestion attempting to make it more clear what it is :)